packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -17,44 +20,72 @@ export const useRequestRequisitionColumns = ({
   sortBy,
   onChangeSortBy,
 }: UseRequestRequisitionColumnOptions): Column<RequestRequisitionLineFragment>[] => {
+  const t = useTranslation('common');
   return useColumns<RequestRequisitionLineFragment>(
     [
-      ['itemCode', { width: 50 }],
-      ['itemName', { width: 150 }],
-
-      'monthlyConsumption',
+      [
+        'itemCode',
+        { width: 100, accessor: ({ rowData }) => rowData.item.code },
+      ],
+      [
+        'itemName',
+        { width: 350, accessor: ({ rowData }) => rowData.item.name },
+      ],
 
       [
-        'previousStockOnHand',
+        'monthlyConsumption',
         {
+          width: 200,
           accessor: ({ rowData }) =>
-            `${rowData.itemStats.availableStockOnHand} (${Math.floor(
-              (rowData.itemStats.availableStockOnHand ?? 0) /
-                (rowData?.itemStats.averageMonthlyConsumption ?? 0)
-            )} months)`,
+            rowData.itemStats.averageMonthlyConsumption,
         },
       ],
-      [
-        'calculatedQuantity',
-        {
-          accessor: ({ rowData }) => {
-            const threeMonthsStock =
-              rowData?.itemStats.averageMonthlyConsumption ?? 1 * 3;
-            const diff =
-              threeMonthsStock - (rowData?.itemStats.availableStockOnHand ?? 0);
-            if (diff > 0) {
-              return `${diff.toFixed(2)} (${Math.floor(
-                diff / (rowData?.itemStats.averageMonthlyConsumption ?? 1)
-              )} months)`;
-            } else {
-              return 0;
-            }
-          },
+
+      {
+        key: 'availableStockOnHand',
+        label: 'label.stock-on-hand',
+        align: ColumnAlign.Left,
+
+        width: 200,
+        accessor: ({ rowData }) => {
+          const { itemStats } = rowData;
+          const { availableStockOnHand, availableMonthsOfStockOnHand } =
+            itemStats;
+
+          const monthsString = availableMonthsOfStockOnHand
+            ? `${availableMonthsOfStockOnHand} ${t('label.months')}`
+            : '';
+          return `${availableStockOnHand} ${monthsString}`;
```

Comment:
should that have brackets around it?
```suggestion
          return `${availableStockOnHand} (${monthsString})`;
```